examples/python/reconstruction_system/scripts/download_redwood_simulated.sh

```diff
@@ -4,9 +4,9 @@ set -e
 download_redwood_scene()
 {
     DATA_NAME=$1
-    wget http://redwood-data.org/indoor/data/$DATA_NAME-color.zip
-    wget http://redwood-data.org/indoor/data/$DATA_NAME-depth-clean.zip
-    wget http://redwood-data.org/indoor/data/$DATA_NAME-depth-simulated.zip
+    wget https://github.com/intel-isl/open3d_downloads/releases/download/redwood-simulated/$DATA_NAME-color.zip
```

Comment:
I would add a comment pointing to the original link. Same for once below.

Reply:
Added.